lib/Echidna/ABI.hs

```diff
@@ -232,7 +233,9 @@ shrinkInt x = fromIntegral <$> getRandomR (0, toInteger x)
 shrinkAbiValue :: MonadRandom m => AbiValue -> m AbiValue
 shrinkAbiValue (AbiUInt n m)         = AbiUInt n <$> shrinkInt m
 shrinkAbiValue (AbiInt n m)          = AbiInt n  <$> shrinkInt m
-shrinkAbiValue x@AbiAddress{}        = pure x
+shrinkAbiValue x@AbiAddress{}        = case x of
```

Comment:
let's put the pattern match in `shrinkAbiValue` and not a separate case

Reply:
Done.